terra-jupyter-aou/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.2.11
+FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.2.12
```

Comment:
I think rebuild `terra-base` automattly upgrades `Cromshell-beta` to `cromshell` per 
https://github.com/broadinstitute/cromshell/releases/tag/2.0.0
Can you help to verify, and if true also remove the alias `Cromshell-beta` code?

Reply:
[This PR](https://github.com/DataBiosphere/terra-docker/pull/424) covers that change.

Reply:
Once that PR is released I'll bump the version and rebase this branch.